Toolchain/BuildClang.sh

```diff
@@ -322,6 +322,12 @@ popd
 
 pushd "$DIR/Local/clang/bin/"
     buildstep "mold_symlink" ln -s ../../mold/bin/mold ld.mold
+
+    for arch in $ARCHS; do
+        ln -s clang "$arch"-pc-serenity-clang
```

Comment:
`buildstep "symlinks"` for all these including mold_symlink above?

Reply:
or remove buildstep from mold_symlink, no real preference from me :man_shrugging: